src/app/core/header/header.component.ts

```diff
@@ -7,12 +9,22 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HeaderComponent implements OnInit {
 
-  constructor() { }
+  private isAuth = false;
+  private userName = '';
+
+  constructor(private authService: AuthorizationService) {
+    this.userName = this.authService.getUserInfo();
+    this.isAuth = this.authService.isAuthentificated();
+    authService.checkAuth.subscribe(value => { 
+      this.isAuth = value.isAuthentificated; 
+      this.userName = value.userName;
+    });
```

Comment:
Please save this subscription in some property and then unsubscribe on destroy phase to increase performance